subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1425,6 +1516,128 @@ static void le_remove_cig(struct net_buf *buf, struct net_buf **evt)
 
 #endif /* CONFIG_BT_CENTRAL */
 
+#if defined(CONFIG_BT_CTLR_CENTRAL_ISO) || \
+	defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
```

Comment:
Missing broadcaster and sync checks here as well I suppose

Reply:
I will leave this to a later PR